sdk/appcenter/src/main/java/com/microsoft/appcenter/ingestion/OneCollectorIngestion.java

```diff
@@ -117,6 +134,35 @@ public ServiceCall sendAsync(String appSecret, UUID installId, LogContainer logC
         }
         headers.put(API_KEY, apiKey.toString());
 
+        /* Gather tokens from logs. */
+        JSONObject tickets = new JSONObject();
+        for (Log log : logContainer.getLogs()) {
+            List<String> ticketKeys = ((CommonSchemaLog) log).getExt().getProtocol().getTicketKeys();
+            if (ticketKeys != null) {
+                for (String ticketKey : ticketKeys) {
+                    String token = TicketCache.getInstance().getTicket(ticketKey);
+                    if (token != null) {
+                        try {
+                            tickets.put(ticketKey, "d:" + token);
```

Comment:
Why do we add the `d:` prefix to the token?

Reply:
It's from email discussion, the protocol documents `"p:"` instead but seems like bearer token is actually `"d:"`. There are prefixes for each type.